src/NLSModels.jl

```diff
@@ -9,6 +9,24 @@ export AbstractNLSModel, nls_meta, NLSCounters, reset!,
 
 abstract type AbstractNLSModel <: AbstractNLPModel end
 
+function show(io :: IO, nls :: AbstractNLSModel)
```

Comment:
I think this is the `show` for a `NLSModelMeta`.

Reply:
This uses information from `nls.meta` and `nls.nls_meta`. I can redefine `NLSMeta` to keep all `meta` information as well. It would be a huge change, though, but it can be done.

Reply:
What I mean is that this method could just call `show(nls.meta)` and also `show(nls.nls_meta)`.

Reply:
By separating, I think I'll have to drop the 3-column format.